src/Microsoft.Graph.Core/Models/AsyncOperationStatus.cs

```diff
@@ -2,42 +2,38 @@
 //  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
 // ------------------------------------------------------------------------------
```

Comment:
Git decided this was a rename and I didn't catch it. Deleting and re-adding the file didn't fix it so I gave up.